Azure Security Center - Updated IoT Security Solution

liranc opened the pull request:
Relevant spec update: https://github.com/Azure/azure-rest-api-specs/pull/7283

Commit:
Regenerated Microsoft.Security

Commit:
Updated IoT recordings

Commit:
Merge branch 'master' of https://github.com/Azure/azure-sdk-for-net

isra-fel reviewed (changes requested):
Hi @liranc , could you
- Write tests for the new APIs
- Update versions if you are going to release

Thanks

Commit:
Bumped version to 1.0.2.0

Commit:
Added SubAssessmentTests

isra-fel commented:
@liranc if you re adding new feature please update minor version, and if it involves breaking change please update the major version.

Also the version in `.csproj` needs to be updated.
Thanks

Commit:
Updated version to 1.1.0

liranc commented:
> @liranc if you re adding new feature please update minor version, and if it involves breaking change please update the major version.
> 
> Also the version in `.csproj` needs to be updated.
> Thanks

This update is backward compatibility and only adds functionality.
I've reverted my previous version change and now I've just updated the minor version, so we're at `1.1.0`.

Thanks
